README.md

```diff
@@ -161,8 +178,9 @@ Unit's output should contain both snippets, neatly organized:
 }
 ```
 
-For full details of configuration management, see the
-[docs](https://unit.nginx.org/configuration/#configuration-management).
+## WebAssembly
+Unit supports deploying WASM as a supported application.
+For more information see [the docs](https://unit.nginx.org/configuration/#configuration-wasm).
```

Comment:
Line break after `see`. Or `the` and don't include it in the link (for consistency with the others)...

Reply:
It's generally bad practice not to specify where a link goes:
❌: "go here" , "see the doc", "check this"
✔: "check the wasm configuration documentation", "check the Microsoft Azure docs", etc

I would change this to `For more information see the [Unit configuration docs]`